.github/workflows/photonvision-docs.yml

```diff
@@ -0,0 +1,45 @@
+name: PhotonVision Sphinx Documentation
+
+on:
+  push:
+    branches: [ master ]
+    paths:
+      - 'docs/**'
+      - '.github/**'
+  pull_request:
+    branches: [ master ]
+    paths:
+      - 'docs/**'
+      - '.github/**'
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/checkout@v3
+
+    - uses: actions/setup-python@v4
+      with:
+        python-version: '3.9'
+
+    - name: Install and upgrade pip
+      run: python -m pip install --upgrade pip
+
+    - name: Install Python dependencies
+      working-directory: docs
+      run: |
+        pip install sphinx sphinx_rtd_theme sphinx-tabs sphinxext-opengraph doc8
+        pip install -r requirements.txt
+
+    - name: Check links
+      working-directory: docs
+      run: make linkcheck || true
```

Comment:
is this so link check doesnt stop the build? i believe the right thing here would be continue-on-error

https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#jobsjob_idstepscontinue-on-error

Reply:
oh yeah